coalib/output/ConsoleInteraction.py

```diff
@@ -23,6 +23,28 @@
     RESULT_SEVERITY, RESULT_SEVERITY_COLORS)
 from coalib.settings.Setting import Setting
 
+from pygments import highlight
+from pygments.formatters import (TerminalTrueColorFormatter,
+                                 TerminalFormatter)
+from pygments.filters import VisibleWhitespaceFilter
+from pygments.lexers import TextLexer, get_lexer_for_filename
+from pygments.style import Style
+from pygments.token import Token
+from pygments.util import ClassNotFound
+
+
+class BackgroundSourceRange(Style):
+    styles = {
+        Token: 'bold bg:#BB4D3E #111'
+    }
+
+
+class BackgroundMessage(Style):
```

Comment:
not sure, maybe add `Style` to the name of these classes? As when you use them the name doesn't really look like a style :3